release_dashboard/lib/services/release_dashboard_start_context.dart

```diff
@@ -3,11 +3,17 @@
 // found in the LICENSE file.
 
 import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_core/proto.dart' as pb;
 import 'package:file/file.dart';
+import 'package:flutter/material.dart';
 import 'package:process/process.dart';
+import 'package:provider/provider.dart';
+
+import '../state/status_state.dart';
 
 class ReleaseDashboardStartContext extends StartContext {
   ReleaseDashboardStartContext({
+    required this.context,
```

Comment:
A widget context is necessary for the state real-time updater. The updater could be called with `context.read<StatusState>().syncStatusWithState();`